swagger.yaml

```diff
@@ -1,20 +1,238 @@
+      tags:
+        - home
+      responses:
+        '200':
+          content: 
+            'application/json':
+              schema: 
+                type: object
+                properties:
+                  account:
+                    description: The admin/super account information
+                    type: object
+                    properties:
+                      balance:
+                        description: The super/admin account remaining balance in XDC
+                        type: string
+                      walletAddress:
+                        description: The wallet address of the account
+                        type: string
+                    required:
+                      - walletAddress
+                      - balance
+                  relayer:
+                    type: object
+                    properties:
+                      backlog:
+                        description: The current gap between audited block in smartcontract (parent chain) and latest minded block in subnet 
+                        type: integer
+                      contractAddress:
+                        type: string
+                    required:
+                      - contractAddress
+                      - backlog
+                required:
+                  - account
+                  - relayer
+        '400':
+          description: 'Bad Request'
+        '409':
+          description: 'Conflict'
+        '500':
+          description: 'Server Error'
+  /information/network:
+    get:
+      summary: Get the subnet network information, such as block mining time, tx throughput etc
+      tags:
+        - home
+      responses:
+        '200':
+          content: 
+            'application/json':
+              schema: 
+                type: object
+                properties:
+                  subnet:
+                    type: object
+                    properties:
+                      block:
+                        description: block metadata, such as mining frequency
+                        type: object
+                        properties:
+                          averageBlockTime:
+                            description: The block mining time per X second.
+                            type: integer
+                          txThroughput:
+                            description: The subnet transaction throughput, we only keep track of the last 10 txs and take average per second.
+                            type: integer
+                    required:
+                      - block
+                  parentChain:
+                    type: object
+                    properties:
+                      targetChain:
+                        description: A string value which is used to identify the target parent chain. It can be a URL, IP address or a name.
+                        type: string
+                    required:
+                      - targetChain
```

Comment:
I think we have flat format:
{
   averageBlockTime
   txThroughput
   targetChain
}

Reply:
nah. there are mixed information here. some belong to subnet and some pointing to the actual parent chain network. In the case of adding more fields in the future, i recommend keep it with nested layer.